nifi-nar-bundles/nifi-influxdb-bundle/nifi-influxdb-processors/src/test/java/org/apache/nifi/processors/influxdb/ITExecuteInfluxDBQuery.java

```diff
@@ -110,8 +125,72 @@ public void testCreateDB() {
         assertEquals("Value should be equal",null, flowFiles.get(0).getAttribute(ExecuteInfluxDBQuery.INFLUX_DB_ERROR_MESSAGE));
         assertEquals("Value should be equal",query, flowFiles.get(0).getAttribute(ExecuteInfluxDBQuery.INFLUX_DB_EXECUTED_QUERY));
 
-        flowFiles.get(0).assertContentEquals("{\"results\":[{}]}");
+        QueryResult queryResult = gson.fromJson(new StringReader(new String(flowFiles.get(0).toByteArray())), QueryResult.class);
+        assertEquals("results array should be empty", 1, queryResult.getResults().size());
```

Comment:
I think it would be a good addition to have: `assertNotNull(queryResult.getResults())` before this.

Reply:
Corrected.